README.md

```diff
@@ -221,6 +221,12 @@ The design is available in open source in the Figma file format under the URL ht
 ## Other platforms
 Vue Storefront is platform agnostic which means it can be connected to virtually any CMS. Please take a look at [Pimcore bridge](https://github.com/DivanteLtd/coreshop-vsbridge) to give you an idea of how other platforms can be connected. Any support for integrating Prestashop, Shopify ... - much appreciated.
 
+## Concerns when hosting
+When hosting VueStorefront there are some challenges, especially around NodeJS, you should be aware of.
+Server Side Rendering can suffer from memory leaks because of suboptimal code. More on how to avoid these for VueJS can be ready in [this article](https://vuejs.org/v2/cookbook/avoiding-memory-leaks.html). We also recommend reading about [VueJS best practices](https://blog.usejournal.com/vue-js-best-practices-c5da8d7af48d).
+
+ On the server we advice to run [PM2](http://pm2.keymetrics.io/) which offers features to keep your NodeJS application stable. When hosting on Kubernetes the checks and memory limits can be leveraged to kill unhealthy containers.
+
```

Comment:
Make it more positive please as we disscussed + linking to Cloud as a solution + pointing out the docs: https://docs.vuestorefront.io/guide/installation/production-setup.html